### PR TITLE
ADEN-5136 add adonis-marker regardless of detection result

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-5136

### CODE REVIEW

File: extensions/wikia/AdEngine/js/AdEngine2.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ define('ext.wikia.adEngine.adEngine', [
 	'ext.wikia.adEngine.utils.hooks',
 	'wikia.document',
 	'wikia.lazyqueue',
-	'wikia.log'
+	'wikia.log',
+	require.optional('ext.wikia.aRecoveryEngine.pageFair.recovery')
 ], function (
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/AdEngine2.js]: MAJOR Function has 11 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/provider/gpt/helper.js [hunk 2]
```diff
@@ -13,8 +13,7 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 	'ext.wikia.aRecoveryEngine.adBlockRecovery',
 	'ext.wikia.adEngine.slotTweaker',
 	'wikia.log',
-	require.optional('ext.wikia.adEngine.provider.gpt.sraHelper'),
-	require.optional('ext.wikia.aRecoveryEngine.pageFair.recovery')
+	require.optional('ext.wikia.adEngine.provider.gpt.sraHelper')
 ], function (
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js]: MAJOR Function has 13 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues
* ![MAJOR][MAJOR] 7 major
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 7 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Remove this unused "$wgUser" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L47](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L47): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [helper.js#L55](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L55): Function has a complexity of 14 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [AdEngine2.decorators.spec.js#L117](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/AdEngine/js/spec/AdEngine2.decorators.spec.js#L117): Remove the unused function parameter "decoratedFillInSlot". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![INFO][INFO] [AdEngine2.js#L64](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/AdEngine/js/AdEngine2.js#L64): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [AdEngine2.js#L87](https://github.com/Wikia/app/blob/e6274f4d8f144112ad91fd21ffa7584ae1752458/extensions/wikia/AdEngine/js/AdEngine2.js#L87): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues
* ![MAJOR][MAJOR] 7 major
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 7 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Remove this unused "$wgUser" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L47](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L47): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [helper.js#L55](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L55): Function has a complexity of 14 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [AdEngine2.decorators.spec.js#L117](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/AdEngine/js/spec/AdEngine2.decorators.spec.js#L117): Remove the unused function parameter "decoratedFillInSlot". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![INFO][INFO] [AdEngine2.js#L64](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/AdEngine/js/AdEngine2.js#L64): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [AdEngine2.js#L87](https://github.com/Wikia/app/blob/b2e8b548a81c5f8c58a122f119703f8ab73dd04f/extensions/wikia/AdEngine/js/AdEngine2.js#L87): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues
* ![MAJOR][MAJOR] 7 major
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 7 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Remove this unused "$wgUser" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L47](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L47): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [helper.js#L55](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L55): Function has a complexity of 14 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [AdEngine2.decorators.spec.js#L117](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/AdEngine/js/spec/AdEngine2.decorators.spec.js#L117): Remove the unused function parameter "decoratedFillInSlot". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![INFO][INFO] [AdEngine2.js#L64](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/AdEngine/js/AdEngine2.js#L64): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [AdEngine2.js#L87](https://github.com/Wikia/app/blob/d66febd01ac65d5261d85dd3a2bfed9efada0876/extensions/wikia/AdEngine/js/AdEngine2.js#L87): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Remove this unused "$wgUser" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L21](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L21): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [ARecoveryModule.class.php#L47](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L47): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2.js#L13](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/AdEngine2.js#L13): Function has 10 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [helper.js#L18](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L18): Function has 14 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [helper.js#L57](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L57): Function has a complexity of 16 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [AdEngine2.decorators.spec.js#L117](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/spec/AdEngine2.decorators.spec.js#L117): Remove the unused function parameter "decoratedFillInSlot". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![INFO][INFO] [AdEngine2.js#L62](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/AdEngine2.js#L62): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [AdEngine2.js#L85](https://github.com/Wikia/app/blob/ae2e0d7095c897ad7afd8ca4ffd215b3fb85ba04/extensions/wikia/AdEngine/js/AdEngine2.js#L85): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by v3rth — APPROVED** [review]

**Pull request merged**